packages/dropdowns.next/src/types/index.ts

```diff
@@ -239,7 +222,7 @@ export interface IMenuListProps extends HTMLAttributes<HTMLUListElement> {
   /** Determines menu expansion */
   isExpanded?: boolean;
   /** Provides a list of acceptable fallback placements */
-  fallbackPlacements?: GardenPlacement[];
+  fallbackPlacements?: Exclude<GardenPlacement, 'auto'>[];
```

Comment:
`'auto'` does not make sense (and is not accepted by Floating UI) as a fallback placement.